OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -528,6 +548,41 @@ public bool CanRepairAt(Actor host)
 			return repairable != null && repairable.Info.RepairActors.Contains(host.Info.Name) && self.GetDamageState() != DamageState.Undamaged;
 		}
 
+		bool IsBlockedBy(Actor self, Actor otherActor, Actor ignoreActor)
+		{
+			// We are not blocked by the actor we are ignoring.
+			if (otherActor == ignoreActor)
```

Comment:
If you change this to `if (otherActor == self || otherActor == ignoreActor)` you can drop the work-arounds...

Reply:
Yep, I already did this is #16417. But it makes more sense to include it here. Changed.